header.php

```diff
@@ -135,13 +135,23 @@
       }
 ?>
 <?php
-    // if is page or is people archive
-      if (is_page() || is_post_type_archive('people')) {
+    // if is page
+      if (is_page()) {
+        global $post;
+?>
+          <a href="<?php echo home_url('profile/'); ?>"><li <?php is_active_page($post->ID); ?>>Profile</li></a>
+          <a href="<?php echo home_url('news/'); ?>"><li <?php is_active_page($post->ID); ?>>Clients</li></a>
+          <a href="<?php echo home_url('awards/'); ?>"><li <?php is_active_page($post->ID); ?>>Awards</li></a>
+          <a href="<?php echo home_url('people/'); ?>"><li>People</li></a>
+          <a href="<?php echo home_url('recruit/'); ?>"><li <?php is_active_page($post->ID); ?>>Recruit</li></a>
+<?php
+    // or is people archive
+      } else if (is_post_type_archive('people')) {
 ?>
           <a href="<?php echo home_url('profile/'); ?>"><li>Profile</li></a>
           <a href="<?php echo home_url('news/'); ?>"><li>Clients</li></a>
           <a href="<?php echo home_url('awards/'); ?>"><li>Awards</li></a>
-          <a href="<?php echo home_url('people/'); ?>"><li>People</li></a>
+          <a href="<?php echo home_url('people/'); ?>"><li class="font-color-active">People</li></a>
           <a href="<?php echo home_url('recruit/'); ?>"><li>Recruit</li></a>
```

Comment:
why not be conditional only to the people li item? the rest of the list could be the same code, no?